include/net/wifi_mgmt.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief WiFi L2 stack public header
+ */
+
+#ifndef __WIFI_MGMT_H__
+#define __WIFI_MGMT_H__
+
+#include <net/net_mgmt.h>
+
+/* Management part definitions */
+
+#define _NET_WIFI_LAYER	NET_MGMT_LAYER_L2
```

Comment:
Since an offload driver would bypass L2, I wonder if this definition has any significance?   Is NET_MGMT_LAYER actually used anywhere?

Reply:
design wise, the wifi mgmt interface is located in L2. And in the future, if zephyr gets a 802.11 Soft MAC, it will be in L2 as well (the same interface will work there)